txnkv/transaction/2pc.go

```diff
@@ -919,7 +922,10 @@ const CommitSecondaryMaxBackoff = 41000
 
 // doActionOnGroupedMutations splits groups into batches (there is one group per region, and potentially many batches per group, but all mutations
 // in a batch will belong to the same region).
-func (c *twoPhaseCommitter) doActionOnGroupMutations(bo *retry.Backoffer, action twoPhaseCommitAction, groups []groupedMutations) error {
+func (c *twoPhaseCommitter) doActionOnGroupMutations(
+	bo *retry.Backoffer,
+	action twoPhaseCommitAction, groups []groupedMutations,
```

Comment:
format looks weird.

Reply:
I believe this was for wrapping long lines by IDE. But it should not affect unchanged lines.